src/plugins/section/schema.js

```diff
@@ -20,7 +20,11 @@ function normalizeDocument(change, error) {
         if (child.type === 'title') {
             change.setNodeByKey(child.key, { type: 'paragraph' })
             return
+        } else if (child.type === 'invalid') {
+            change.removeNodeByKey(child.key)
+            return
         }
```

Comment:
What is this for?

Reply:
I guess I've added this while debugging. I'll delete this.